docs/WorkflowGuide.md

```diff
@@ -14,17 +14,18 @@ nav-text: Workflow Guide
 
 ## Table of Contents
 
-- [Introduction](#introduction)
-- [Forking Workflow](#forking-workflow)
-- [Issues](#issues)
-- [Pull Requests](#pull-requests-pr)
-- [Reviews](#reviews)
-- [Unfinished Work](#unfinished-work)
-- [Pre commit githook](#pre-commit-githook)
+- [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Forking Workflow](#forking-workflow)
+  - [Issues](#issues)
+  - [Pull Requests (PR)](#pull-requests-pr)
+  - [Reviews](#reviews)
+  - [Unfinished Work](#unfinished-work)
+  - [Pre commit githook](#pre-commit-githook)
 
 ### Introduction
 
-imPoster Workflow Guide documents the team's agreed upon standardisation of
+imPoster Workflow Guide documents the team's agreed upon standardization of
```

Comment:
Would the use of American spelling be better? I think it is mentioned somewhere in textbook